docs/pages/nextjs.page.server.mdx

```diff
@@ -86,7 +86,7 @@ Next.js aggressively uses [caching](https://nextjs.org/docs/app/building-your-ap
 
 ### Server-first mentality
 
-Next.js - made by Vercel, a hosting provider - is server-focused. It now defaults to interpreting all React components as React Server Components, so you have to mark React components with "use client" to opt-out of always hitting the server. This can reduce the bundle size for a lot of regular web-sites, as the client doesn't need to download JS libraries that now by default run on the server (e.g. markdown). But leaning more on the server may not be what you want.
+Next.js - made by Vercel, a hosting provider - is server-focused. It now defaults to interpreting all React components as React Server Components, so you have to mark React components with "use client" to opt-out of always hitting the server. While it reducess the side of client-side bundles, as the client doesn't need to download JS libraries that now by default run on the server (e.g. markdown), it leans more on the server which may not be what you want.
```

Comment:
Hm.. if we join the sentence like this then it will have 40 words. I try to avoid sentences with more than around 20 words. Since they become hard to read. By the end of the sentence I've forgotten the start of the sentence. 😅  Especially if it references something in the start, and there's a lot of text in between.

Reply:
👍 Done 3304c856cc96d1104989945ab8c55b17ff2ca053. (Feel free to directly push such improvements.)